auth/cloud-client/snippets_test.py

```diff
@@ -31,3 +32,12 @@ def test_explicit():
 
     with mock.patch('io.open', open_mock):
         snippets.explicit()
+
+
+def test_explicit_compute_engine():
+    adc, project = google.auth.default()
+    credentials_patch = mock.patch(
```

Comment:
Because splitting context manager syntax over multiple lines is *extremely* awkward, so we've gotten into the habit of giving the patch manager its own variable.